pkgs/development/python-modules/markdown-it-py/default.nix

```
@@ -26,16 +26,16 @@
 
 buildPythonPackage rec {
   pname = "markdown-it-py";
-  version = "3.0.0";
+  version = "4.0.0";
   format = "pyproject";
```

```suggestion
  pyproject = true;
```
and replace `nativeBuildInputs` with `build-system`, etc.